instruqt-tracks/terraform-basics-aws/track.yml

```diff
@@ -0,0 +1,472 @@
+slug: terraform-basics-aws
+id: 3grwcipacthi
+type: track
+title: Terraform Basics on AWS
+teaser: Learn how to begin provisioning infrastructure on AWS with Terraform.
+description: |-
+  Terraform is the world's most popular open source provisioning tool. Thousands of users build millions of containers, virtual machines, and other infrastructure with Terraform every single day. Terraform is easy to learn and fun to work with.
+
+  We invite you to join the ranks of the Terraform community today!
+
+  This track will introduce you to Terraform open source on AWS. You'll learn  terraform setup, basic commands and integration with AWS.
+icon: https://storage.googleapis.com/instruqt-hashicorp-tracks/logo/terraform.png
+tags:
+- terraform
+- aws
+- basics
+owner: hashicorp
+developers:
+- go@hashicorp.com
+private: true
+published: true
+challenges:
+- slug: setup-our-environment
+  id: xywjxcqb096l
+  type: challenge
+  title: "\U0001F3E1 Moving in - Set Up Your Workspace"
+  teaser: Configure your code editor with the HCL extension and open a workspace.
+  assignment: |-
+    **Goal**: Validate and configure your working environment.
+    **Estimated Time**: 2-3 minutes.
```

Comment:
Put these on separate lines using a `<br>` tag at the end of the first line.

Reply:
Done.